runtime/onert/frontend/circle_traininfo/include/traininfo_loader.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __CIRCLE_TRAININFO_LOADER_H__
+#define __CIRCLE_TRAININFO_LOADER_H__
+
+#include "ir/train/TrainingInfo.h"
+#include "ir/Model.h"
+
+namespace onert
+{
+namespace traininfo_loader
+{
+
+static constexpr const char *TRAININFO_METADATA_NAME = "CIRCLE_TRAINING";
+
+std::unique_ptr<ir::train::TrainingInfo> loadTrainingInfo(const uint8_t *buffer, const size_t size);
```

Comment:
I have a plan to call this interface, while loading a model file (`nnfw_session::load_model_from_..`)

This function will be used like here : 
https://github.com/Samsung/ONE/pull/12152/files#r1426184845